lib/parser/objects.ts

```diff
@@ -215,6 +215,7 @@ export class OGenericConstant extends OGeneric implements I.IVariableBase, I.IHa
 }
 export class OReference extends ObjectBase implements I.IHasDefinitions, I.IHasReferenceToken {
   definitions: ObjectBase[] = [];
+  aggregate = false; // if this reference is an aggregate
```

Comment:
How can a reference be a aggregate? I dont unterstand? an aggregate aggregates multiple names  or?